docs/1.0. EDID-Mapping.md

```diff
@@ -0,0 +1,220 @@
+# AMWA BCP-XXX-YY: EDID Mapping in NMOS Systems \[Work In Progress\]
+
+
+## Background
+
+AMWA NMOS specifications provide resources for specifying receiver capabilities of an NMOS device.  Recent extension of these resources allow NMOS Controllers developed for PRO A/V User-Stories to be created.  In particular, these NMOS Controllers include Controllers developed for the AIMS sponsered IPMX set of open standards [IPMX](#ipmx-ref). 
+
+## Scope
+
+This document provides Best Current Practice guideluines for how to implement mappings of EDID elements onto AMWA IS-04 and IS-EDID-CM resources.  The document also provides guidance on how implementers may use the APIs to achieve some common IPMX type User-Stories and associated use-cases as described in the EDID Connection Management Architecture [EDID-CM-ARCH](#edid-cm-arch-ref).
```

Comment:
> how to implement mappings of EDID elements onto AMWA IS-04 and IS-EDID-CM resources

It looked to me that the main purpose of the BCP is mapping EDID to BCP-004-01 Receiver Caps. What am I missing?